crypto/keys/secp256k1/eth/secp256k1.go

```diff
@@ -0,0 +1,211 @@
+package eth
```

Comment:
considering the crypto package structure I don't think this should be a package inside `secp256k1`, I would create a `ethsecp256k1`  package.

Reply:
Sounds good.